cron/poll_pull_requests.py

```diff
@@ -73,6 +73,16 @@ def poll_pull_requests(api):
             # the PR is mitigated or the threshold is not reached ?
             if variance >= threshold or not is_approved:
                 voting_window = gh.voting.get_extended_voting_window(api, settings.URN)
+
+                # read the db to see if this PR is expedited by /vote fast...
+                try:
+                    issue = Issue.get(issue_id=pr["id"])
+                    if issue.expedited: # expedite
+                        voting_window /= 2
```

Comment:
This code needs to be moved out of the if statement for the extended voting window. Currently this just halves the extended voting window.